content/docs/command-reference/commit.md

```diff
@@ -20,9 +20,9 @@ positional arguments:
 The `dvc commit` command is useful for several scenarios, when data already
 tracked by DVC changes: when a [stage](/doc/command-reference/run) or
 [pipeline](/doc/command-reference/dag) is in development/experimentation; when
-manually editing or generating DVC <abbr>outputs</abbr>; or to force update the
-`dvc.lock` or `.dvc` files without reproducing stages or pipelines. These
-scenarios are further detailed below.
+force updating the `dvc.lock` or `.dvc` files without reproducing stages or
+pipelines; or when manually editing or generating DVC <abbr>outputs</abbr> or
+adding them to a stage. These scenarios are further detailed below.
```

Comment:
> or when manually editing or generating DVC <abbr>outputs</abbr> or
adding them to a stage

To many "or"s and the order is inverted compared to the bullet list below. But let's combine them conceptually. Generalize. Basically you just removed the comma so far 🙂  How can you describe all the cases of the last sentence in a single statement? Should be possible, I think.

Reply:
Tried combine the last point in same sentence in the latest update. Please check.